ibis/backends/pandas/execution/window.py

```diff
@@ -53,13 +62,18 @@ def _post_process_group_by(
     parent: pd.DataFrame,
     order_by: List[str],
     group_by: List[str],
+    **kwargs,
```

Comment:
Should this be timecontext instead of kwargs?

Reply:
It could be, but `timecontext` is currently not used in this function so I didn't make it as an explicit param

Reply:
I think it's a good pattern to specify all param explicitly even some of them are not used. kwargs brings flexibility which I don't think you need it here.

Reply:
(Explicitness over implicitness)

Reply:
agree with @icexelloss here, we don't want kwargs at all.

Reply:
thx, I removed `kwargs`